build.gradle.kts

```diff
@@ -19,7 +19,7 @@ dependencies {
         exclude(module = "opus-java")
     }
 
-    implementation("de.chojo", "cjda-util", "2.0.3-alpha.5")
+    implementation("de.chojo", "cjda-util", "2.1.1+alpha.9")
```

Comment:
The + confuses me, is this the version we plan to use in the next time @RainbowDashLabs ?

Reply:
Yes that is the current latest stable version of cjda.
Renovate couldnt detect newer versions when suffixed with a - in the end. That was an attempt to fix it since + is the actual correct marker for a free suffix. However it still doent really work and still confuses renovate :D Might need to take a deeper look there.